mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala

```diff
@@ -227,10 +227,10 @@ class AFTSurvivalRegression @Since("1.6.0") (@Since("1.6.0") override val uid: S
 
     if (handlePersistence) instances.unpersist()
 
-    val coefficients = Vectors.dense(weights.slice(2, weights.length))
-    val intercept = weights(1)
-    val scale = math.exp(weights(0))
-    val model = new AFTSurvivalRegressionModel(uid, coefficients, intercept, scale)
+    val regressionCoefficients = Vectors.dense(coefficients.slice(2, coefficients.length))
```

Comment:
Sorry for late attention. I think we should not rename ```coefficients``` to ```regressionCoefficients``` and ```weights``` to ```coefficients```, because the original ```coefficients``` is correctly used and should not renamed.
When I was writing ```AFTSurvivalRegression```, I have noticed the misusage of ```weights``` and use the right name ```coefficients```. But the ```weights``` is misused and should be renamed, because that it's different from other ```weights```, it include three parts : log(sigma), intercept and coefficients. I have send #9431 to rename ```weights``` to ```parameters``` or other names which will not bring confusion, it's not a public variable. And rename ```regressionCoefficients``` back to ```coefficients```.